protos/sources/vega/events/v1/events.proto

```diff
@@ -13,6 +13,45 @@ import "vega/vega.proto";
 
 option go_package = "code.vegaprotocol.io/vega/protos/vega/events/v1";
 
+// Event notifying the details of a funding interval of a perpetuals market
+message FundingPeriod {
+  // ID of the market for which this funding period relates to.
+  string market_id = 1;
+  // Sequence number of the funding period
+  uint64 seq = 2;
+  // Time in Unix nanosecond when the funding period started.
+  int64 start = 3;
+  // Time in Unix nanosecond when the funding period ended.
```

Comment:
Will this only ever be seen in the past?

Reply:
This will only be set if the funding period has ended, so yes